ui/app/AppLayouts/Profile/views/AdvancedView.qml

```diff
@@ -455,6 +455,17 @@ SettingsContentBase {
                     Global.openPopup(enableAutoMessageConfirmationDialogComponent)
                 }
             }
+
+            StatusSettingsLineButton {
+                anchors.leftMargin: 0
+                anchors.rightMargin: 0
+                text: qsTr("Fake loading screen")
+                isSwitch: true
+                switchChecked: root.advancedStore.isFakeLoadingScreenEnabled
```

Comment:
hmmm maybe this should not be visible in production?

Reply:
Yeah definitely pls